cpp/src/ray/runtime/abstract_ray_runtime.cc

```diff
@@ -53,6 +52,16 @@ std::shared_ptr<AbstractRayRuntime> AbstractRayRuntime::DoInit() {
   } else {
     ProcessHelper::GetInstance().RayStart(TaskExecutor::ExecuteTask);
     runtime = std::shared_ptr<AbstractRayRuntime>(new NativeRayRuntime());
+    auto redis_ip = ConfigInternal::Instance().redis_ip;
+    if (redis_ip.empty()) {
+      redis_ip = GetNodeIpAddress();
+    }
+    std::string redis_address =
+        redis_ip + ":" + std::to_string(ConfigInternal::Instance().redis_port);
+    runtime->global_state_accessor_ =
+        ProcessHelper::GetInstance().CreateGlobalStateAccessor(
```

Comment:
Can we move this logic into NativeRayRuntime? "CreateGlobalStateAccessor" only works in native mode.